data/json/itemgroups/custom_lbvs.json

```diff
@@ -88,5 +96,12 @@
     "//": "stuff that might be in a molle pack",
     "subtype": "collection",
     "entries": [ { "item": "camelbak", "prob": 40 }, { "group": "molle_accessories", "count": [ 1, 2 ] } ]
+  },
+  {
+    "type": "item_group",
+    "id": "molle_belt_contents",
+    "//": "stuff that might be clipped to a molle webbing belt",
+    "subtype": "collection",
+    "entries": [ { "group": "molle_accessories", "count": [ 1, 2 ] } ]
```

Comment:
you should def make a custom list of items "that might be clipped to a molle webbing belt" molle_accessories even has a chance to spawn a deployment bag and other kind of large accessories

Reply:
I seperated it into two itemgroups, one for bag/vests and one for belts but both drawing from molle_accessories, and moved deployment_bag to backpack only, what do you think?

Also I saw your previous pr, this should mostly shift the chance of spawning holsters to the belt, and reduces the likelihood of multiple holsters spawning on a single piece of gear.

Reply:
lgtm, if i've missed something we can always adjust the itemgroups in followup